examples/pysp/scripting/apps/compile_scenario_tree.py

```diff
@@ -102,7 +102,7 @@ def _pickle_compiled_scenario(worker,
             param._validate = None
         for set_ in block.component_objects(Set):
             set_.initialize = None
-            set_.filter = None
+            #set_.filter = None
```

Comment:
Can this commented line be removed?

Reply:
Yes, but I wanted to leave it in for the review so that @ghackebeil, @jwatsonnm, and @DLWoodruff would see it and could comment.

Reply:
Feel free to delete. This code was just about removing initialization stuff that no longer seemed to be needed once a model becomes "static".